code/modules/admin/verbs/selectequipment.dm

```diff
@@ -0,0 +1,197 @@
+/client/proc/cmd_select_equipment(mob/M in GLOB.mob_list)
+	set category = "Admin.Events"
+	set name = "Select equipment but better"
+
+
+	var/datum/select_equipment/ui  = new(usr, M)
+	ui.ui_interact(usr)
+
+/datum/select_equipment
+	var/client/user
+	var/mob/target
+
+	var/dummy_key
+	var/mob/living/carbon/human/dummy/dummy
+
+	var/static/list/cached_outfits
+
+	//normally a path; an initialized outfit object if it's a custom outfit
+	var/datum/outfit/selected_outfit = /datum/outfit
+	//used to keep track of which outfit the UI has selected
+	var/selected_name = "/datum/outfit"
+
+/datum/select_equipment/New(usr, mob/M)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names